libs/shared/metrics/glean/src/registry/subplat-backend-metrics.yaml

```diff
@@ -1,3 +1,390 @@
+    expires: never
+    data_sensitivity:
+      - interaction
+
+  source:
+    description: |
+      The source from where the user started.  For example, if the user clicked on a
+      link on the Mozilla accounts web site, this value could be 'fx-website'.  The
+      value has a max length of 128 characters with the alphanumeric characters, _
+      (underscore), forward slash (/), . (period), % (percentage sign), and -
+      (hyphen) in the allowed set of characters.
+    type: string
+    lifetime: application
+    notification_emails:
+      - subplat-team@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7531
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=tbd
+    expires: never
+    data_sensitivity:
+      - interaction
+
+  term:
+    description: |
+      This metric is similar to the `utm.source`; it is used in the Firefox
+      browser.  For example, if the user started from about:welcome, then the value
+      could be 'aboutwelcome-default-screen'.  The value has a max length of 128
+      characters with the alphanumeric characters, _ (underscore), forward slash (/),
+      . (period), % (percentage sign), and - (hyphen) in the allowed set of
+      characters.
+    type: string
+    lifetime: application
+    notification_emails:
+      - subplat-team@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7531
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=tbd
+    expires: never
+    data_sensitivity:
+      - interaction
+
+pay_setup:
+  view:
+    description: Checkout view event
+    type: event
+    notification_emails:
+      - subplat-team@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7531
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=tbd
+    expires: never
+    data_sensitivity:
+      - interaction
+  engage:
+    description: Checkout engage event
+    type: event
+    notification_emails:
+      - subplat-team@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7531
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=tbd
+    expires: never
+    data_sensitivity:
+      - interaction
+  submit:
+    description: Checkout submit event
+    type: event
+    notification_emails:
+      - subplat-team@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7531
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=tbd
+    expires: never
+    data_sensitivity:
+      - interaction
+  success:
+    description: Checkout engage event
+    type: event
+    notification_emails:
+      - subplat-team@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-7531
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=tbd
+    expires: never
+    data_sensitivity:
+      - interaction
+  fail:
+    description: Checkout fail event
```

Comment:
Quick drive-by comment - for some of our events like failures, FxA adds a 'reason' ([see example](https://github.com/mozilla/fxa/blob/f39d66b07fac16ea34875d061b10192ec5fd4e8a/packages/fxa-settings/src/pages/Signin/utils.ts#L172)). Not sure if you want this too.

Reply:
As part of the SubPlat P1 Metrics epic from last year, the `reason` field was actually renamed to `error_id` for the SubPlat events. Not sure exactly what the motivation was for that change, but I figured we'd stick with that. [SubPlat P1 Product Metrics - FxA Epic Technical Specification](https://docs.google.com/document/d/1-KdXzZbn3Utw2aekJbcc1lrMV9tNb19tzifPbdtQaso/edit)